luthier/config/src/tables.lua

```diff
@@ -168,15 +167,26 @@ M.physical = {
 
 M.logical = {
     wikipedia = {
+        type = "abridged",
         metrics =  {"count", "added", "delta", "deleted"},
         dimensions = wikipedia_dimensions,
-        granularity = {"all", "hour", "day"},
-        physicalTables = {"wikiticker"}
+        granularities = {
+            --"all",
```

Comment:
Why is all commented out?

Reply:
I have not implemented the casing on all, yet. I will start working on that now.

Reply:
Use https://github.com/gab-umich/fili/blob/logicalTable/fili-core/src/main/java/com/yahoo/bard/webservice/data/time/AllGranularity.java for the all granularity.

Reply:
This is now done. 
Also adapted the granularity dictionary idea.
Especially, when we build the LIP without going through the `LuthierBinderFactory > getConfigurationLoader()`, it will not load in `logicalTableGroups` due to the fact that we have no granularity dictionary. 
Instead of throwing a null-pointer error, I will instead check if the granularity dictionary exists. 
This leads to three possible cases: 

- granularityDictionary does not exist: will not load in any `logicalTable`; log a warning in using logger; proceed without exception. (If I throw exception, it will break all previous unit tests.)
- granularityDictionary exists but it does not contain the needed granularity key (String): will throw LuthierFactoryException: this means that the user clearly wants to configure the `LogicalTable`, but failed to do so by not supporting their custom time grain. From the end result, it looks harsher than the previous one. 
- granularityDictionary exists, contains needed String (lower case, including 'all'), we load in `logicalTable` as needed, proceed without exception.

Reply:
I know we have a conversation about building the default granularity dictionary in the builder.  If so we shouldn't have to handle a null granularity dictionary unless someone does it on purpose.

Reply:
And it doesn't look done, because all is still commented out.

Reply:
Ah I forgot to push my commits. It will go live in a few minutes.

Reply:
And for the issue you raise, I believe we do have to handle that just to let the previously passing unit tests (which are all built directly using the `LuthierIndustrialPark.Builder` thus has a `null` as their granularity), keep passing. 
If we do not special case that, then we have to

- somehow populate the granularity, which requires _modifying the previous unit tests_ (maybe using a withGranularityDictionary(), which I also happen to have coded) OR
- make the `LuthierIndustrialPark.load()` not fail when missing the entire granularityDictionary, which is effectively what I am trying to accomplish, but just on the factory level, instead of on the LIP level.

Reply:
I'd like to hear your suggestions, we can also examine the new commits together.